src/test/java/org/testobject/api/TestObjectClientTest.java

```diff
@@ -61,7 +54,6 @@ public void testStartQualityReport() {
 
 	@Test @Ignore
```

Comment:
i think we should either remove these tests or implement them properly. they are not used anyway so what's the point in having them?

Reply:
from first look they are useless, but I see in history that people even added more tests with "ignore" annotation.
Check this link https://github.com/saucelabs/testobject-java-api/commit/210c139b069007f9c19d3b4c43514d8ffc51eeac